eslint/eslint-plugin-packlets/src/PackletAnalyzer.ts

```diff
@@ -75,12 +84,32 @@ export class PackletAnalyzer {
 
     // Example: /path/to/my-project/src
     let srcFolderPath: string | undefined;
+    // Example: /path/to/my-project/tsconfig.json
+    const tsconfigFilePath: string | undefined = compilerOptions['configFilePath'] as string;
 
     if (!tsconfigFilePath) {
       this.error = { messageId: 'missing-tsconfig' };
       return;
     }
 
+    // Construct the path alias tester
+    // Example { "compilerOptions": { "paths": { "@my-org/*": ['app/*'], "@tools": ['tools'] } } }
+    const tsconfigPaths: MapLike<string[]> = compilerOptions.paths ? compilerOptions.paths : {};
+    // Remove any path mappings which are pointing to src/packlets, or node_modules
+    // (this would be nice to make configurable)
+    // Example [[alias, paths], ...]
+    const nonPackletMappings: [string, string[]][] = Object.entries(tsconfigPaths).filter(
+      ([, paths]) =>
+        !paths.some((path) => path.startsWith('src/packlets') || path.startsWith('node_modules/'))
```

Comment:
Should we consider other equivalent forms such as `./src/packlets` instead of `src/packlets`?

Reply:
Also IIRC `baseUrl` affects the interpretation of these paths.  (If so it would be okay to simply report an error if `baseUrl` is present rather than trying to implement the full semantics.)